docs/_news/news07.md

```diff
@@ -20,14 +20,16 @@ Curious? [Learn more](remoteoverview.html)!
 
 #### New Features and Highlights for 0.7.0
 
+**Monitoring Updates**
+- You can now [inject metrics](https://github.com/codewind-resources/design-documentation/blob/master/codewindServer/metricsInjection.md) into your Node.js, Liberty, and Spring projects. 
+- Effortlessly [visualize data](https://github.com/eclipse/codewind/issues/977) like CPU useage, memory footprint, and web traffic on the included application monitoring dashboard at the click of a button. If you don't want it or need it, simply [disable it](https://github.com/eclipse/codewind/issues/1290). Codewind hosts the dashboard, so displaying the graphs won't eat into your project's resources.
```

Comment:
Oops, didn't catch this typo earlier. Sorry about that! "useage" needs to be "usage".

Reply:
committed new change